### ISSUE TITLE
Brave Ads not shown on News feed 

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#20412]
## Description 
Found while testing #20011, Brave Ads not shown on News feed 

## Steps to reproduce 

   1. Enable Brave News via flag
   2. Opt-in to Brave News
   3. No ads are shown 

## Actual result 
Brave Ads not shown on News feed 

```

2022-01-10 01:58:18.073 2527-2527/? V/chromium: [VERBOSE1:bundle.cc(128)] Creative set id 461d9877-85e0-40fd-8685-964898762cac does not support android
2022-01-10 01:58:18.073 2527-2527/? V/chromium: [VERBOSE1:bundle.cc(428)] creative set id 461d9877-85e0-40fd-8685-964898762cac has no entries
2022-01-10 01:58:18.073 2527-2527/? V/chromium: [VERBOSE1:bundle.cc(128)] Creative set id 2f05daed-4062-4be9-b4a8-a5459ea72f11 does not support android
2022-01-10 01:58:18.073 2527-2527/? V/chromium: [VERBOSE1:bundle.cc(428)] creative set id 2f05daed-4062-4be9-b4a8-a5459ea72f11 has no entries
2022-01-10 01:58:18.075 2527-2527/? V/chromium: [VERBOSE1:bundle.cc(128)] Creative set id 4e2fe7f1-d9a7-4b7d-988f-ab3e0bcabec3 does not support android
2022-01-10 01:58:18.075 2527-2527/? V/chromium: [VERBOSE1:bundle.cc(428)] creative set id 4e2fe7f1-d9a7-4b7d-988f-ab3e0bcabec3 has no entries
2022-01-10 01:58:18.076 2527-2527/? V/chromium: [VERBOSE1:bundle.cc(128)] Creative set id 05fa6b16-f12a-44ee-9914-1e8ab99aab93 does not support android
2022-01-10 01:58:18.076 2527-2527/? V/chromium: [VERBOSE1:bundle.cc(128)] Creative set id 05fa6b16-f12a-44ee-9914-1e8ab99aab93 does not support android
2022-01-10 01:58:18.076 2527-2527/? V/chromium: [VERBOSE1:bundle.cc(428)] creative set id 05fa6b16-f12a-44ee-9914-1e8ab99aab93 has no entries
2022-01-10 01:58:18.077 2527-2527/? V/chromium: [VERBOSE1:bundle.cc(128)] Creative set id 0e2869a9-b40f-455d-a599-76c2e6bc332e does not support android
2022-01-10 01:58:18.077 2527-2527/? V/chromium: [VERBOSE1:bundle.cc(428)] creative set id 0e2869a9-b40f-455d-a599-76c2e6bc332e has no entries
2022-01-10 01:58:18.117 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(634)] Successfully saved creative promoted content ads state
2022-01-10 01:58:18.117 2527-2527/? V/chromium: [VERBOSE2:epsilon_greedy_bandit_resource.cc(50)] Successfully loaded epsilon greedy bandit segments:
2022-01-10 01:58:18.118 2527-2527/? V/chromium: [VERBOSE2:epsilon_greedy_bandit_resource.cc(52)]   untargeted
2022-01-10 01:58:18.118 2527-2527/? V/chromium: [VERBOSE2:epsilon_greedy_bandit_resource.cc(52)]   Technology & Computing
2022-01-10 01:58:18.118 2527-2527/? V/chromium: [VERBOSE2:epsilon_greedy_bandit_resource.cc(52)]   Real Estate
2022-01-10 01:58:18.118 2527-2527/? V/chromium: [VERBOSE2:epsilon_greedy_bandit_resource.cc(52)]   Personal Finance
2022-01-10 01:58:18.119 2527-2527/? V/chromium: [VERBOSE2:epsilon_greedy_bandit_resource.cc(52)]   Business
2022-01-10 01:58:18.119 2527-2527/? V/chromium: [VERBOSE2:epsilon_greedy_bandit_resource.cc(52)]   Crypto Curious
2022-01-10 01:58:18.119 2527-2527/? V/chromium: [VERBOSE2:epsilon_greedy_bandit_resource.cc(52)]   Consumer Crypto
2022-01-10 01:58:18.119 2527-2527/? V/chromium: [VERBOSE2:epsilon_greedy_bandit_resource.cc(52)]   crypto
2022-01-10 01:58:18.119 2527-2527/? V/chromium: [VERBOSE2:epsilon_greedy_bandit_resource.cc(52)]   Crypto Purchase Intent
2022-01-10 01:58:18.119 2527-2527/? V/chromium: [VERBOSE2:epsilon_greedy_bandit_resource.cc(52)]   Arts & Entertainment
2022-01-10 01:58:18.120 2527-2527/? V/chromium: [VERBOSE2:epsilon_greedy_bandit_resource.cc(52)]   Hobbies & Interests
2022-01-10 01:58:18.120 2527-2527/? V/chromium: [VERBOSE2:epsilon_greedy_bandit_resource.cc(52)]   Intel B2B
2022-01-10 01:58:18.122 2527-2527/? V/chromium: [VERBOSE1:ad_server.cc(158)] Fetch catalog in 2 hours, 16 minutes, 56 seconds at 2:45 PM
2022-01-10 01:58:18.122 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(470)] Successfully deleted creative ad notifications state
2022-01-10 01:58:18.123 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(482)] Successfully deleted creative inline content ads state
2022-01-10 01:58:18.123 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(494)] Successfully deleted creative new tab page ads state
2022-01-10 01:58:18.124 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(506)] Successfully deleted creative new tab page ad wallpapers state
2022-01-10 01:58:18.133 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(518)] Successfully deleted creative promoted content ads state
2022-01-10 01:58:18.140 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(530)] Successfully deleted campaigns state
2022-01-10 01:58:18.149 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(542)] Successfully deleted segments state
2022-01-10 01:58:18.157 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(554)] Successfully deleted creative ads state
2022-01-10 01:58:18.165 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(566)] Successfully deleted dayparts state
2022-01-10 01:58:18.173 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(578)] Successfully deleted geo targets state
2022-01-10 01:58:18.246 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(592)] Successfully saved creative ad notifications state
2022-01-10 01:58:18.258 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(606)] Successfully saved creative inline content ads state
2022-01-10 01:58:18.273 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(620)] Successfully saved creative new tab page ads state
2022-01-10 01:58:18.274 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(646)] Successfully purged expired conversions
2022-01-10 01:58:18.282 2527-2527/? V/chromium: [VERBOSE3:bundle.cc(658)] Successfully saved conversions state
2022-01-10 01:58:21.621 2527-2527/? V/chromium: [VERBOSE1:text_classification_model.cc(90)] No text classification probabilities found for en-US locale
2022-01-10 01:58:21.621 2527-2527/? V/chromium: [VERBOSE1:eligible_inline_content_ads_v1.cc(40)] Get eligible inline content ads:
2022-01-10 01:58:21.630 2527-2527/? V/chromium: [VERBOSE1:eligible_inline_content_ads_v1.cc(150)] Get eligible ads for untargeted segment
2022-01-10 01:58:21.636 2527-2527/? V/chromium: [VERBOSE1:eligible_inline_content_ads_v1.cc(161)] No eligible ads for untargeted segment
2022-01-10 01:58:21.636 2527-2527/? V/chromium: [VERBOSE1:inline_content_ad_serving.cc(81)] Inline content ad not served: No eligible ads found
2022-01-10 01:58:25.277 2527-2527/? V/chromium: [VERBOSE1:rewards_service_impl.cc(466)] Starting ledger process
```
## Expected result
Should show ads similar to desktop 

## Issue reproduces how often 
Easy

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? NA
- Can you reproduce this issue with the current Play Store Beta version? Yes
- Can you reproduce this issue with the current Play Store Nightly version?Yes

## Device details
   - Install type (ARM, x86): ARM
   - Device type (Phone, Tablet, Phablet): All
   - Android version: All
   
## Brave version
1.35.77 x64 Beta & 1.36.23 x64 Nightly build

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards?NA
- Is the issue reproducible on the latest version of Chrome? NA

### Additional information

cc: @alexsafe @mattmcalister @tmancey @brave/legacy_qa 
Note: issue happens with or without rewards being enabled.

**Comment by anthonypkeane** [comment]
@srirambv  to recheck on Nightly

**Comment by tmancey** [comment]
@anthonypkeane @srirambv https://github.com/brave/brave-browser/issues/20658 should resolve this issue. We will test once the fix is in place. Thanks

**Comment by tmancey** [comment]
Can we please close this as a duplicate of https://github.com/brave/brave-browser/issues/20658
